src/iconsManifest/supportedExtensions.ts

```diff
@@ -82,7 +82,13 @@ export const extensions: IFileCollection = {
     { icon: 'ai2', extensions: ['ai'], format: FileFormat.svg, disabled: true },
     {
       icon: 'al',
-      extensions: [],
+      extensions: ['al'],
+      languages: [languages.al],
+      format: FileFormat.svg,
+    },
+    {
+      icon: 'al_dal',
+      extensions: ['dal'],
```

Comment:
In this case, as we want to enforce the usage of `al_dal` icon regardless of the language id, remove the `languages` line.